packages/main/src/CalendarHeader.js

```diff
@@ -53,6 +62,14 @@ const metadata = {
 		isMonthButtonHidden: {
 			type: Boolean,
 		},
+
+		/**
+		 * @type {boolean}
+		 * @private
+		 */
+		_isSecondaryCalendarType: {
```

Comment:
Instead of adding this property, you can create a getter which returns ```!!secondaryCalendarType``` and use it

Reply:
Done